app/models/piece.rb

```diff
@@ -1,2 +1,8 @@
 class Piece < ActiveRecord::Base
+  belongs_to :game
+
+  def self.types
+    %w(King Queen Rook Bishop Knight Pawn)
```

Comment:
got it from that link you providided in trello: http://samurails.com/tutorial/single-table-inheritance-with-rails-4-part-1/

seemed to make sense.